src/boardlib/api/aurora.py

```diff
@@ -470,10 +469,14 @@ def combine_ascents_and_bids(ascents_df, bids_summary, db_path, grades_dict, gra
             (bids_summary['angle'] == ascent_angle)
         ]
         
+        climb_angle_uuid = f"{ascent_climb_uuid}-{ascent_angle}"
```

Comment:
I think it's worth adding a comment here as it is a little inobvious why this would be necessary for Boardlib:

```suggestion
        # Used for Climbdex to uniquely identify climbs at a particular angle
        climb_angle_uuid = f"{ascent_climb_uuid}-{ascent_angle}"
```